kolibri/plugins/user/assets/src/vue/sign-in-page/index.vue

```diff
@@ -1,16 +1,93 @@
 <template>
 
   <div>
-    <h1>Sign In</h1>
-    <p>Use this page for in-progress component prototyping.</p>
+    <img class="logo" alt="logo">
+    <h1>{{ $tr('signInInstantSchools') }}</h1>
```

Comment:
We should keep this generic in the Kolibri repo. We will make instant schools specific adjustments elsewhere.